#ifndef STANDARD_PARAMETER_NAME_H
#define STANDARD_PARAMETER_NAME_H

#define   ROSPARAM_NAME_STEP_SIZE   "step_size"
#define   ROSPARAM_NAME_MAX_X       "max_x"
#define   ROSPARAM_NAME_MIN_X       "min_x"
#define   ROSPARAM_NAME_START_X     "start_x"
#define   ROSPARAM_NAME_MAX_Y       "max_y"
#define   ROSPARAM_NAME_MIN_Y       "min_y"
#define   ROSPARAM_NAME_START_Y     "start_y"
#define   ROSPARAM_NAME_GROUP_NAME  "group_name"
#define   ROSPARAM_NAME_STARTPOSE_JOINTSTATE "start_pose_joint_states"

#endif
